### ISSUE TITLE
Feature request: allow dynamic urls

### ISSUE THREAD

**Issue by fruitl00p** [area17/blast#114]
Currently we're bulding a static version of the storybook (`blast:publish`) and including that in a docker image to be deployed to various urls... Thus the STORYBOOK_SERVER_URL still includes the `storybook_preview` (which is weird as i'd like to have a static build of the entire storybook)

So what am i doing wrong? Is this something inside storybook or in the combination of blast and storybook ?

**Comment by ifox** [comment]
Blast is using Storybook Server, so that's completely expected. If you are not using story controls, I understand why you'd think that it could just generate everything statically for publishing, but the moment you have something dynamic in a story, Blast is making a request to your Laravel application to get that Blade view rendered with the new parameters. So technically if you wanted all that to work statically, you would need to generate the static response for every single permutation of your stories arguments. Hope that makes sense.

**Comment by fruitl00p** [comment]
Not really. I can understand that it still requires some backend processing, but using relative urls would be enough. As we publish the 'static' files during CI/CD into a docker image the actual deployment of the resulting containers determine the full-url... Is there any way to allow relative URL's to be enough?
(its not like we're deploying to HTML only server environment)
